http/rest-client-reactive/src/main/resources/certificates/generate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I like that you added this script, but if I get it right, this is script that based on `cert_validity` and `from` was used to generate `ancient-` and `future-` and `outdated-` and the ones without prefix (`keystore.p12`). It would take me a while before I'd understand what was values you used to generate these ks/ts. I'd have to inspect existing stores and deduce it from there, if for example I wanted to fix that OpenShift test. Or in case there this test wasn't working in FIPS-enabled environment (just an example why I'd need it). Can you just add README with commands you used to generate these ts/ks?